frontend/src/App.tsx

```diff
@@ -143,8 +144,9 @@ export const App: React.FC<AppProps> = ({ userConfig, appConfig }) => {
       layout: getSerializedLayout(),
     })
       .then(() => {
-        if (showToast) {
+        if (userInitiated) {
           toast({ title: "Notebook saved" });
+          formatAll(updateCellCode);
```

Comment:
@mscolnick Check config to make sure `format_on_save` is enabled?

Reply:
oh yea, duh, will add